jwst/lib/reffile_utils.py

```diff
@@ -312,6 +217,10 @@ def get_subarray_model(sci_model, ref_model):
     sub_model: JWST data model
         subarray version of the reference file model
     """
+    # If science data is in multistripe readout, use
+    # multistripe-specific subarray reconstruction.
+    if getattr(sci_model.meta.subarray, 'multistripe_reads1') is not None:
```

Comment:
As written, this is equivalent to `sci_model.meta.subarray.multistripe_reads1`.  Did you mean to add a default value to the getattr, in case it's missing?
```suggestion
    if getattr(sci_model.meta.subarray, 'multistripe_reads1', None) is not None:
```